### PR TITLE
Added game criteria to site with a link in the navbar

### PR DESCRIPTION
**Pull request by Dylemperor**

### CODE REVIEW

**Review by kylew1208 — APPROVED** [review]
Looks good

File: BucStop/Views/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -55,7 +56,7 @@
     <!-- This block handles the footer, which includes the version number for the project. -->
     <footer class="border-top footer text-muted">
         <div class="container">
-            &copy; 2023 - BucStop - <a asp-area="" asp-controller="Home" asp-action="Privacy">Privacy</a> - <a href="https://github.com/chrisseals98/BOBBY/releases/tag/v2.0.0" style="color:#C6AA76;">Version: 2.0.0</a>
+            &copy; 2023 - BucStop - <a asp-area="" asp-controller="Home" asp-action="Privacy">Privacy</a> - <a href="https://github.com/chrisseals98/BOBBY/releases/tag/v2.0.0" style="color:#C6AA76;">Version: 2.0.0</a><a> - </a><a asp-area="" asp-controller="Home" asp-action="GameCriteria">Game Criteria</a>
```
Comment (chrisseals98) [inline BucStop/Views/Shared/_Layout.cshtml]: I think a link here is good, but I think it would also be good to have a link to the page on the navbar as well. What do you think?

File: BucStop/compose-dev.yaml
Comment (chrisseals98) [inline BucStop/compose-dev.yaml:1]: Why was this added?

File: BucStop/Views/Shared/_Layout.cshtml (hunk 1 above)
Comment (Dylemperor) [reply BucStop/Views/Shared/_Layout.cshtml]: I think it could be in the footer or the navbar. Probably not both though. If we think the navbar would be better I can change it to go there instead.

File: BucStop/compose-dev.yaml
Comment (Dylemperor) [reply BucStop/compose-dev.yaml:1]: Good question. I don't know why this is here. 
Update: this was auto-generated, so I removed it from the pull request.

**Review by klucher — APPROVED** [review]
Sweet
